publication-commons/src/main/java/no/unit/nva/publication/model/storage/ResourceDao.java

```diff
@@ -187,76 +189,70 @@ public List<TicketDao> fetchAllTickets(AmazonDynamoDB client) {
                    .map(item -> parseAttributeValuesMap(item, TicketDao.class))
                    .collect(Collectors.toList());
     }
-    
+
     @JsonProperty(RESOURCES_BY_CRISTIN_ID_INDEX_PARTITION_KEY_NAME)
     public String getResourceByCristinIdentifierPartitionKey() {
         return extractCristinIdentifier().isEmpty() ? null
-                   : CRISTIN_IDENTIFIER_INDEX_FIELD_PREFIX + KEY_FIELDS_DELIMITER + extractCristinIdentifier();
+                   : CRISTIN_IDENTIFIER_INDEX_FIELD_PREFIX
+                     + KEY_FIELDS_DELIMITER
+                     + extractCristinIdentifier().orElseThrow();
```

Comment:
@truhacevkir I hope moving from CristinIdentifier:Optional[10000026] to CristinIdentifier:10000026 in DB does not imply problems in import :-)